### DISCUSSION TITLE
Raspotify lirc IR control

### DISCUSSION THREAD

**Discussion by jackobtone** [dtcooper/raspotify#483] [Ideas]
For many Raspberry Pi users it would be a gamechanger. Many "store products" have such functionallity, and actually its the only thing that I miss in my RPi transport. Lirc enables me to switch song, pause/stop/play titles from my own disc library using physical remote control, but unfortunatelly I don`t have any idea how to make it work with Raspotify. It would be awesome if someone work it out.

**Comment by JasonLG1979** [comment]
Raspotify is always remotely controlled by whatever Spotify app. That's kinda the point behind Spotify Connect?

**Comment by JasonLG1979** [comment]
And also since Raspotify is just a packaged version of librespot the feature would need to be added up stream.
